addons/onboarding/static/src/views/form/onboarding_step_form_controller.js

```diff
@@ -52,4 +57,13 @@ export default class OnboardingStepFormController extends FormController {
     get stepConfig() {
         return { reloadAlways: false, reloadOnFirstValidation: false };
     }
+    /**
+	 * Returns whether the step condition is met or not after
+     * record is saved
+	 * 
```

Comment:
Normally all the comment markers ("*") have to be on the same indentation level, just like the method here above.

Reply:
My bad, my editor use tab not 4 space :((